File: t/samples/cyclical_graph/c/f.cc
#include "f.h"
#include "c.h"
#include <iostream>

F::F(char* name) {
  this->_name = name;
}

const char* F::name() {
  C* c = new C("Letter C");
  std::cout << c->name() << std::endl;
  return this->_name;
}

File: t/samples/cyclical_graph/csharp/F.cs
using System;
using c;

namespace f {

  public class F {
    private string _name;

    public F(string name) {
      this._name = name;
    }

    public string name() {
      C cprint = new C("Letter C");
      Console.WriteLine(cprint.name());
      return this._name;
    }
  }

}
